### PR TITLE
Django Views ACL. Added App structure.

### PR DESCRIPTION
**Pull request by ntodosiychuk**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+Apache license
```
Comment (pzdeb) [inline LICENSE]: License file must contain full License content, like this: http://choosealicense.com/licenses/mit/

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    long_description=README,
+    classifiers=[
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache License', # example license
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+        # Replace these appropriately if you are stuck on Python 2.
+        'Programming Language :: Python :: 3',
```
Comment (pzdeb) [inline setup.py:24]: Will it work for Python2?

Comment (ntodosiychuk) [inline setup.py:24]: I hope...it should ;)

Comment (ntodosiychuk) [inline setup.py:24]: In the best case with python3 and python2

Comment (pzdeb) [inline setup.py:24]: Must work with both :-)

**Comment by ntodosiychuk** [comment]
@pzdeb @pivancy Code is updated according to your comments

**Pull request merged**